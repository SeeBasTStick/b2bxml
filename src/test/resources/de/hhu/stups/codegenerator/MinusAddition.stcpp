int main() {
    Minus minus;
    minus.calculate();
    cout << minus.getRes();
    return 0;
}