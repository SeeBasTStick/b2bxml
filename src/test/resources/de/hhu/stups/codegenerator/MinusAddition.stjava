public static void main(String[] args) {
    Minus minus = new Minus();
    minus.calculate();
    System.out.println(minus.getRes());
}