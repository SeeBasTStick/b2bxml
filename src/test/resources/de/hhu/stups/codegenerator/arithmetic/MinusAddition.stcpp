int main() {
    Minus arithmetic;
    arithmetic.simulate();
    cout << arithmetic.getValue();
    return 0;
}