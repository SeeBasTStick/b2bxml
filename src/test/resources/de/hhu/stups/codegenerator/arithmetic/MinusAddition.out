public static void main(String[] args) {
    Minus arithmetic = new Minus();
    arithmetic.simulate();
    System.out.println(arithmetic.value);
}